frontend/src/routes/user/page.tsx

```diff
@@ -14,7 +14,7 @@ import { PageFallback } from '../fallbacks/PageFallback.tsx';
  */
 export interface User {
   name: string;
-  given_name: string;
+  givenName: string;
```

Comment:
it should be given_name, the same as in JWT

Reply:
I change it and then you changed it back

Reply:
Okay, thanks, we also caught the snake_case in our types.ts that we did not want so we thought that was the case here too, will fix that asap